bin/dev/create_helm_charts_pr.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is used to open a PR against this repository:
+# https://github.com/suse/kubernetes-charts-suse-com
+# It will fetch the bundle specified by CAP_BUNDLE and will update the helm
+# charts in the above repo.
+
+set -e
+
+if [ -z $GITHUB_USER  ]; then
+  echo "GITHUB_USER environment variable not set"
+  exit 1
+fi
+
+if [ -z $GITHUB_PASSWORD ]; then
+  echo "GITHUB_PASSWORD environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${CAP_BUNDLE}" ]]; then
+  echo "CAP_BUNDLE not set"
+  exit 1
+fi
+
+TMP=$(mktemp -d -p $PWD)
+#trap "rm -rf ${TMP}" EXIT
+
+pushd $TMP
+
+# Get the "hub" cli
+# https://hub.github.com/hub.1.html
+wget https://github.com/github/hub/releases/download/v2.3.0-pre10/hub-linux-amd64-2.3.0-pre10.tgz
+tar xvzf hub-linux-*.tgz
+cp hub-linux-*/bin/hub hub
+HUB=$(readlink -f hub)
```

Comment:
Wait, what? The symlink in the tar file comes broken (because it points into a subdirectory that didn't exist in there)?

```bash
$ tar tvf hub-linux-amd64-2.3.0-pre10.tgz --wildcards '*/bin/hub'
-rwxr-xr-x travis/travis 9396782 2017-06-26 11:35 hub-linux-amd64-2.3.0-pre10/bin/hub
```

It looks like a standard file to me. Might want to rephrase this bit as
```bash
curl -L https://github.com/github/hub/releases/download/v2.3.0-pre10/hub-linux-amd64-2.3.0-pre10.tgz | tar xvz --wildcards --strip-components=2 '*/bin/hub'
HUB="${PWD}/hub"
```

Reply:
`readlink -f` was one way to get the absolute path to the file (since it is not in the $PATH to call it directly). You solution is more elegant. Changing.